templates/docs/examples/patterns/code-snippet/code-snippet.html

```diff
@@ -0,0 +1,29 @@
+{% extends "_layouts/examples.html" %}
+{% block title %}Code Snippet{% endblock %}
+
```

Comment:
It's the time to add standalone stylesheet for code snippets, as standalone examples for this component are rendered unstyled https://vanilla-framework-3474.demos.haus/docs/examples/standalone/patterns/code-snippet/code-snippet.